### DISCUSSION TITLE
Whatsapp audios

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#566] [Q&A]
Hello, 
I processed a UFDR report from Cellebrite with the help of IPED.
After that I realized there is some Whatsapp audios from the extraction, but I can't play them.
Someome knows how to do it so?
Do I have to convert it? Or there is a media player who can play it?
I tested VLC but nothing

thx

**Comment by lfcnassif** [comment]
Currently iped has no video/audio player embedded, there is an open issue to integrate one: #144.

But you should be able to listen to audios externally with simple double clicking. If VLC didn't play the audios, maybe they are corrupted or are just empty references to files in the cloud not recovered. Check if the audios play fine from CellebriteReader software, eventually they could be exported wrongly...

**Reply by thiagocaparelli** [reply]
It was a text file containing just metadata. 
I've found the original audio with the .opus format.
Thank you so much.
